### DISCUSSION TITLE
runAfterGenerator not invoked after storeTranslationsToDisk

### DISCUSSION THREAD

**Discussion by artola** [ivanhofer/typesafe-i18n#579] [Q&A]
I am using `storeTranslationsToDisk` to import json files, and it is not triggering the `runAfterGenerator` even if internally it calls:

https://github.com/ivanhofer/typesafe-i18n/blob/9f21f1217bd0c25466e137ebc78093122769f026/packages/importer/src/importer.mts#L108

And this will only happens if `generateTypes`  is set to `true`, otherwise it returns early:

https://github.com/ivanhofer/typesafe-i18n/blob/9f21f1217bd0c25466e137ebc78093122769f026/packages/importer/src/importer.mts#L86

What I would like to achieve is to run "prettier" (`yarn prettier ./src/i18n --write`) every time a files are created, regardless of types' generation.

**Comment by ivanhofer** [comment]
@artola thanks for your feedback.
I've only added it for the CLI because there you can't control when the files are beeing generated. When you implement `storeTranslationToDisk`, you have control over it, sou you can execute any code afterwards.
But also makes sense to have a consistent behavior and adding it to `storeTranslationToDisk` makes sense here. I'll try to add [this](https://github.com/ivanhofer/typesafe-i18n/blob/9f21f1217bd0c25466e137ebc78093122769f026/packages/generator/src/generator.mts#L97) soon.

**Reply by ivanhofer** [reply]
I've just released version `5.21.0`. Let me know if it works.
